aws/resource_aws_appsync_resolver_test.go

```diff
@@ -98,6 +98,32 @@ func TestAccAwsAppsyncResolver_DataSource(t *testing.T) {
 	})
 }
 
+func TestAccAwsAppsyncResolver_DataSource_lambda(t *testing.T) {
+	var resolver appsync.Resolver
+	rName := fmt.Sprintf("tfacctest%d", acctest.RandInt())
+	resourceName := "aws_appsync_resolver.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSAppSync(t) },
```

Comment:
Required `ErrorCheck` after original submission:

```suggestion
		PreCheck:     func() { testAccPreCheck(t); testAccPreCheckAWSAppSync(t) },
		ErrorCheck:   testAccErrorCheck(t, appsync.EndpointsID),
```